ibis/backends/tests/test_window.py

```diff
@@ -689,17 +689,25 @@ def test_percent_rank_whole_table_no_order_by(backend, alltypes, df):
 
 
 @pytest.mark.notimpl(["dask", "datafusion", "polars"])
+@pytest.mark.broken(["pandas"], reason="pandas returns incorrect results")
```

Comment:
I'm going to leave this marked as broken for now, in the interest of merging the PR. I'll create a follow up issue for it.